src/PhpSpec/Process/ReRunner/WindowsPassthruReRunner.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of PhpSpec, A php toolset to drive emergent
+ * design by specification.
+ *
+ * (c) Marcello Duarte <marcello.duarte@gmail.com>
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace PhpSpec\Process\ReRunner;
+
+use PhpSpec\Process\Context\ExecutionContextInterface;
+use Symfony\Component\Process\PhpExecutableFinder;
+
+class WindowsPassthruReRunner extends PhpExecutableReRunner
+{
+    /**
+     * @var ExecutionContextInterface
+     */
+    private $executionContext;
+
+    /**
+     * @param PhpExecutableFinder $phpExecutableFinder
+     * @param ExecutionContextInterface $executionContext
+     * @return static
+     */
+    public static function withExecutionContext(PhpExecutableFinder $phpExecutableFinder, ExecutionContextInterface $executionContext)
+    {
+        $reRunner = new static($phpExecutableFinder);
+        $reRunner->executionContext = $executionContext;
+
+        return $reRunner;
+    }
+
+    /**
+     * @return boolean
+     */
+    public function isSupported()
+    {
+        return (php_sapi_name() == 'cli')
+            && $this->getExecutablePath()
+            && function_exists('passthru')
+            && (stripos(PHP_OS, "win") === 0);
+    }
+
+    public function reRunSuite()
+    {
+        $args = $_SERVER['argv'];
+        $command = $this->buildArgString() . '"' . $this->getExecutablePath().'" '.join(' ', array_map('escapeshellarg', $args));
```

Comment:
ah I missed that you are applying double quotes manually here, which means doing part of the escaping by hand (I'm saying "part of it", because other chars may need escaping too on Windows, but PHP itself does not handle it totally right: https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Process/ProcessUtils.php#L39)

What I don't understand is why you are applying quotes by hand instead of using the escaping (you should indeed not do both)